### PR TITLE
Fixed Hi2isis MRO:ADC_TIMING_SETTING label conversion issue #4290 and Converted Hi2isis App and Tests

### LINKED ISSUE #4290: hi2isis does not properly convert MRO:ADC_TIMING_SETTINGS label from .IMG to .cub file
**ISIS version(s) affected**: at least 3.9+

**Description**  
HiRISE EDR .IMG files have an `MRO:ADC_TIMING_SETTINGS` in their `INSTRUMENT_SETTING_PARAMETERS` group.

When ISIS `hi2isis` runs, the resulting value of `ADCTimingSetting` is always the value `-9999` regardless of the original value in the EDR file.

**How to reproduce**  
Pick any HiRISE EDR, run `hi2isis`.

**Possible Solution**  
I suspect the problem is with `$ISISROOT/appdata/translations/MroHiriseInstrument.trn` whose `ADCTimingSetting` group is looking for the `MRO:ADC_TIMING_SETTING` key which doesn't have the trailing "S" that the key in the PDS label actually has.

I'm not sure if the PDS labels perhaps changed over time (in which case either the singular or plural version of the key should be retained in the translation file) or if this has just always been broken.  I have a copy of the old version of the ISIS data area and it appears that there has been no trailing "S" since hiriseInstrument.trn.2006-03-30 (which was before launch).  I checked an "early" HiRISE EDR, and it has the "S" on the label and indicates that it was created with software from 2006, so this has probably always been a problem.

This means that every cube file `hi2isis` has ever written has a `ADCTimingSetting = -9999`.  Not sure if an "s" should be placed on the end of the label in the cube file or not when this is fixed.  Since it has always been -9999, nobody has *probably* been relying on it, but they might be?

### PR DESCRIPTION
**Pull request by amystamile-usgs**
## Description
Fixed label conversion issue with MRO:ADC_TIMING_SETTING by updating the MroHiRiseInstrument.trn input key from "MRO:ADC_TIMING_SETTING" to "MRO:ADC_TIMING_SETTINGS". I also went through the label inputs and updated Spider Leg Temperatures labels to reflect the EDR labels (they were also reverting to -9999). Converted hi2isis App and add 3 gtests. 

## Related Issue

Fixes #4290 

## Motivation and Context

Updated instrument trn file to correctly convert label values from EDR file, specifically for MRO_ADC_TIMING_SETTING label input.

## How Has This Been Tested?

Tested with 3 gtests

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [x] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by amystamile-usgs** [comment]
I had issues with cropping the test images. Therefore this PR includes the original test images. Not sure if the files are too large or okay as is to push to dev.

File: isis/src/mro/apps/hi2isis/MroHiriseInstrument.trn [hunk 1]
```diff
@@ -701,22 +701,22 @@ End_Group
 
 Group = SpiderLeg120Temperature
```
Comment (scsides) [inline isis/src/mro/apps/hi2isis/MroHiriseInstrument.trn]: Although it is not required, the group names should be changed to keep confusion down.

File: isis/src/mro/apps/hi2isis/MroHiriseInstrument.trn [hunk 2]
```diff
@@ -701,22 +701,22 @@ End_Group
...
   InputGroup     = TEMPERATURE_PARAMETERS
   InputPosition  = TEMPERATURE_PARAMETERS
   InputDefault   = -9999
-  OutputName     = SpiderLeg120Temperature
+  OutputName     = SpiderLeg150Temperature
   OutputPosition = (Object, IsisCube, Group, Instrument)
   Translation    = (*, *)
 End_Group
 
 Group = SpiderLeg240Temperature
```
Comment (scsides) [inline isis/src/mro/apps/hi2isis/MroHiriseInstrument.trn]: ditto

**Review by scsides — DISMISSED** [review]
This should have a change log entry

File: isis/src/mro/apps/hi2isis/MroHiriseInstrument.trn (hunk 1 above)
Comment (amystamile-usgs) [reply isis/src/mro/apps/hi2isis/MroHiriseInstrument.trn]: Good catch. Groups names now updated.

**Comment by amystamile-usgs** [comment]
> This should have a change log entry

Added a change log entry.

**Comment by Kelvinrr** [comment]
@amystamile-usgs I think ~5mb is acceptable, especially considering HiRise sizes, if we have no other choice. I generally trust that this is probably the best we can do. 

One thing I am wondering is that we need a better way to organize new test data in a way that isn't app specific (i.e. group images based on mission). Not something to address here, especially since these dont seem reusable, but something to consider on apps that use cubes.

**Review by scsides — APPROVED** [review]

**Pull request merged**